app_mobilenet_alpha_1/src/support.cpp

```diff
@@ -0,0 +1,70 @@
+#include <platform.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <xcore/channel.h>
+
+#include "xcore_optimised_mobilenetV2_input_160x160x3_alpha_1.tflite.h"
+
+#if defined __GNUC__
+#define ALIGN(X) __attribute__((aligned(X)))
+#elif defined _MSC_VER
+#define ALIGN(X) __declspec(align(X))
+#elif defined __TASKING__
+#define ALIGN(X) __align(X)
+#endif
+
+#ifdef SHARED_MEMORY_ARENA
+uint8_t tensor_arena[162192] ALIGN(8);
+#endif
+
+unsigned char checksum_calc(char *data, unsigned int length) {
+  static char sum;
+  static char *end;
+  sum = 0;
+  end = data + length;
+
+  do {
+    sum -= *data++;
+  } while (data != end);
+  return sum;
+}
+
+void init1(unsigned flash_data) {
+  printf("\nModel1 init");
+  model_init((void *)flash_data);
+  printf("\nModel1 init done");
+}
+
+void run1() {
+  printf("\nModel1 input");
+  int8_t *p = model_input(0)->data.int8;
```

Comment:
Since this is meant to be a sample, please modify this loop to iterate over all model_inputs(). 

`for(int n=0; n< model_inputs(); ++n) {
        int8_t *p = model_input(n)->data.int8;
        // input is simply filled with -128 to 127 repeatedly
        // -128, -125, -122...127
        int k = -128;
        for (int i=0;i<model_input_size(n);++i) {
            if (k >= 128) {
                k = -128;
            }
            p[i] = k;
            k = k + 3;
        }
    }`